dist/yoast-seo.js

```diff
@@ -1194,9 +1248,27 @@ function createDefaultSnippetPreview() {
  * @constructor
  */
 YoastSEO.App = function( args ) {
-	this.config = this.extendConfig( args );
+	if ( !isObject( args ) ) {
+		args = {};
+	}
+	defaultsDeep( args, defaults );
+
+	this.config = args;
 	this.callbacks = this.config.callbacks;
 
+	if ( !isObject( this.callbacks.getData ) ) {
```

Comment:
Please put all the error throwing in a different private function for readability of the constructor.